.github/workflows/deploy.yml

```diff
@@ -0,0 +1,26 @@
+name: Deploy to Skynet
+on:
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: 14.x
+
+      - name: Install dependencies
+        run: npm install
+
+      - name: Build
+        run: npm run build
+
+      - name: Deploy to Skynet
+        uses: SkynetLabs/deploy-to-skynet-action@v1
```

Comment:
I'm not sure @v1 will work, if not then use @main

Reply:
Looks like v1 works